src/databricks/labs/lsql/dashboards.py

```diff
@@ -193,18 +194,47 @@ def split(self) -> tuple[str, str]:
         return "", self._content
 
 
+class FilterHandler(BaseHandler):
+    """Handle filter files."""
+
+    def _parse_header(self, header: str) -> dict:
+        if not header:
+            return {}
+        metadata = json.loads(header)
```

Comment:
i gave it another thought and realised that we're using either `yml`, `sql`, or `md` extensions already. perhaps we should do `.filter.yml` instead of `.filter.json`.

Reply:
Resolved as part of https://github.com/databrickslabs/lsql/pull/290